prometheus-exporters/cloudprober/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: cloudprober
-version: 0.13.0
+version: 0.13.9
```

Since this chart is doing more than just pulling and deploying the image and there are other things, which could be subject to change in the future, I don't want to pin the chart version to the cloudprober version, since this will differ inevitably.

```suggestion
version: 0.13.1
```